packages/gnupg.rb

```diff
@@ -34,20 +34,10 @@ class Gnupg < Package
   depends_on 'sqlite' # R
   depends_on 'zlibpkg' # R
 
-  def self.build
-    system "./configure #{CREW_OPTIONS} \
-              --enable-maintainer-mode \
-              --with-zlib \
-              --with-bzip2 \
-              --with-readline"
-    system "make -j #{CREW_NPROC} || make -j 1"
-  end
+  configure_options '--enable-maintainer-mode \
```

Comment:
Why are we enabling maintainer mode? We normally disable it, if it's not already disabled by default.

Reply:
I don't think it really matters in all practical terms.  It is there to help maintainers.

Reply:
True, but its not useful for our usecase-- it's just an extra line of code.